pkg/subctl/cmd/show_networks.go

```diff
@@ -52,14 +52,26 @@ func showNetwork(cmd *cobra.Command, args []string) {
 }
 
 func showNetworkSingleCluster(config *rest.Config) {
-	dynClient, clientSet, err := getClients(config)
-	exitOnError("Error creating clients for cluster", err)
+	submariner := getSubmarinerResource(config)
+	if submariner != nil {
+		fmt.Println("    Discovering Network details via Submariner:")
+		clusterNetwork := network.ClusterNetwork{
+			PodCIDRs:      []string{submariner.Status.ClusterCIDR},
+			ServiceCIDRs:  []string{submariner.Status.ServiceCIDR},
+			NetworkPlugin: submariner.Status.NetworkPlugin,
+			GlobalCIDR:    submariner.Status.GlobalCIDR,
+		}
+		clusterNetwork.Show()
```

Comment:
Won't this show twice the header?

https://github.com/submariner-io/submariner-operator/blob/ddf805966b110765dbf0d9d5226cced591794d75/pkg/discovery/network/network.go#L46

You need to remove that L46 in network.go and add it before L75 on this file show_networks.go

Reply:
I want to explicitly mention that the details where discovered from Subm CR. The current implemetion result is

```
Discovering Network details via Submariner
Discovered network details (if cn != nil)
else
No network details discovered
```

What you are suggesting might be misleading when cn == nil
The output will be
```
Discovered  network details:
No network details discovered
```

Reply:
Discovering = trying it. Process is in progress.

Discovered is already happened. Saying "No network details discovered" after saying "Discovered  network details:" is conflicting

Reply:
Can you run it with both submariner and not submariner installed and paste the output?

This will always show: fmt.Printf("    Discovered network details:\n")

I think it will be redundant.

```
Discovering Network details via Submariner:
Discovered network details:
```